src/Components/NavigationBar.js

```diff
@@ -88,28 +104,47 @@ function NavigationBar() {
         </div>
         <div className={classes.NavBarMenu}>
           {[
-            { name: "Home", logo: home, link: "home" },
-            {
-              name: "Arbitrage",
-              logo: arbitrage,
-              link: "arbitrage-dashboard"
-            },
-            { name: "Trading", logo: trading, link: "trading-dashboard" },
-            { name: "API", logo: api, link: "api" },
-            { name: "Data", logo: data, link: "data" }
-          ].map(e => (
-            <NavLink to={e.link} className={classes.navBarMenuText}>
-              <div className={classes.navBarMenuTextCursor}>
-                <div className={classes.navBarMenuButton}>
-                  <img
-                    style={{ margin: "0px 10px", marginLeft: 50 }}
-                    src={e.logo}
-                  />
-                  {e.name}
+            { name: "Home", logo: home, link: "home", status: 'inactive' },
+            { name: "Arbitrage", logo: arbitrage, link: "arbitrage-dashboard", status: 'active' },
+            { name: "Trading", logo: trading, link: "trading-dashboard", status: 'active' },
+            { name: "API", logo: api, link: "api", status: 'inactive' },
+            { name: "Data", logo: data, link: "data", status: 'inactive' }
+          ].map(e => 
+            e.status === 'inactive' ? (
+             <AlertDialog
+              button={
+                <NavLink to={'/arbitrage-dashboard'} className={classes.navBarMenuTextDisabled}>
+                  <div className={classes.navBarMenuTextCursor}>
+                    <div className={classes.navBarMenuButton}>
+                      <img
+                        className={classes.navBarMenuIcon}
+                        src={e.logo}
+                      />
+                      {e.name}
+                    </div>
+                  </div>
+                </NavLink>
+              }
+                title={"Coming Soon"}
+                message={`We're still working on features included in the ${e.name} page. Come back soon!`}
+            />
+            ) : (
+             <div style={{ all: 'inherit' }}>
+               <NavLink to={e.link} className={classes.navBarMenuText}>
+                <div className={classes.navBarMenuTextCursor}>
+                  <div className={classes.navBarMenuButton}>
+                    <img
+                      className={classes.navBarMenuIcon}
+                      src={e.logo}
+                    />
+                    {e.name}
+                  </div>
+
                 </div>
-              </div>
-            </NavLink>
-          ))}
+              </NavLink>
+             </div>
+            )  
+          )}
```

Comment:
Like how this indicates in progress nature of this component as it's part of a future release canvas.